packages/shared/lib/core/profile/actions/profiles/checkAndMigrateChrysalisProfiles.ts

```diff
@@ -152,6 +152,9 @@ function isChrysalisProfile(profile: IPersistedProfile | IChrysalisPersistedProf
         if (chrysalisProfileNetworkId) {
             return chrysalisNetworkIdsArray.includes(chrysalisProfileNetworkId)
         }
+        const hasChrysalisNode =
+            chrysalisProfile.settings?.networkConfig?.nodes?.some((value) => isOfficalChrysalisNode(value.url)) || false
```

Comment:
it could be a custom node only, can we maybe asume that if it has any node in this key is actually chrysalis? i dont see this model in stardust so  maybe as soon as an array with an element exists perhaps we can asume is chrysalis